src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,46 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+	public List<Long> sumDigPow(long start, long end) {
```

Comment:
start and end is better than a and b, but still they don't really describe what they are there for